helm/harbor-scanner-aqua/README.md

```diff
@@ -115,8 +115,8 @@ The following table lists the configurable parameters of the scanner adapter cha
 | `scanner.store.redisPoolMaxActive`      | The max number of connections allocated by the pool for a redis store   | `5`            |
 | `scanner.store.redisPoolMaxIdle`        | The max number of idle connections in the pool for a redis store        | `5`            |
 | `scanner.store.redisScanJobTTL`         | The time to live for persisting scan jobs and associated scan reports   | `1h`           |
-| `service.type`                          | Kubernetes service type                                                 | `LoadBalancer` |
-| `service.port`                          | Kubernetes service port                                                 | `8443`         |
+| `service.type`                          | Kubernetes service type                                                 | `ClusterIP`    |
+| `service.port`                          | Kubernetes service port                                                 | `8080`         |
```

Comment:
Did you mean to change the service port & type? I'd have thought that LoadBalancer might be more likely for production use but perhaps not?

Reply:
Yes. This is all deliberate and consistent with what we actually have in the `values.yaml`:
- ClusterIP - it's enough. Typically we don't have to expose the adapter service to the outside world as long as it runs in the same cluster as Harbor.
- By default the TLS is disabled and 8080 is the default port